emails/components/Edit/compareProjects.ts

```diff
@@ -0,0 +1,27 @@
+interface FieldDifferences {
+  [field: string]: boolean;
+}
+export function compareProjects(project: any, requestedProject: any): { [field: string]: boolean } {
+  const fieldsToCompare = [
+    'name',
+    'description',
+    'ministry',
+    'cluster',
+    'projectOwnerId',
+    'primaryTechnicalLeadId',
+    'secondaryTechnicalLeadId',
+    'productionQuota',
+    'testQuota',
+    'developmentQuota',
+    'toolsQuota',
+    'commonComponents',
+  ];
+
+  const differences: { [field: string]: boolean } = {};
+
+  fieldsToCompare.forEach((field) => {
+    differences[field] = JSON.stringify(project[field]) !== JSON.stringify(requestedProject[field]);
```

Comment:
As `objects` keys are not ordered, there is no guarantee that two objects will create the same JSON string.
Let's try using `lodash`'s `isEqual` (or something equivalent) to compare two objects.
https://www.npmjs.com/package/lodash.isequal